ci/windows/test_libcudacxx.ps1

```
@@ -20,16 +20,33 @@ If($CURRENT_PATH -ne "ci") {
+        throw "Finding the producer job failed (exit code $LASTEXITCODE)"
+    }
+    $producerId = "$producerId".Trim()
+    $artifactName = "z_libcudacxx-test-artifacts-$env:DEVCONTAINER_NAME-$producerId"
+    Write-Host "Unpacking artifact '$artifactName'"
+    Invoke-Checked {
+        & bash "./util/artifacts/download_packed.sh" "$artifactName" "../"
+    } "Downloading test artifacts failed"
+} else {
+    $buildCmd = "$PSScriptRoot/build_libcudacxx.ps1 -std $CXX_STANDARD -arch '$CUDA_ARCH' -cmake-options '$CMAKE_OPTIONS'"
+    Write-Host "Running: $buildCmd"
+    Invoke-Expression $buildCmd
```

PSScriptAnalyzer (1.25.0)

[warning] Invoke-Expression is used. Please remove Invoke-Expression from script and find other options instead.

(PSAvoidUsingInvokeExpression)